pkg/metrics/metrics.jsx

```diff
@@ -592,7 +644,18 @@ class CurrentMetrics extends React.Component {
         return (
             <Gallery className="current-metrics" hasGutter>
                 <Card id="current-metrics-card-cpu">
-                    <CardTitle>{ _("CPU") }</CardTitle>
+                    <CardHeader className='align-baseline'>
+                        <CardTitle>{ _("CPU") }</CardTitle>
+                        { !isNaN(this.cpuTemperature) &&
+                        <Flex className="temperature-flex" spaceItems={{ default: 'spaceItemsXs' }}>
+                            <FlexItem className={this.cpuTemperatureColors.iconColor}>
```

Comment:
Should we really change the text color here? Red is still okay, but yellow-on-white is getting into poor contrast territory. Coloring the icon seems enough to me? @garrett